src/webapp/components/survey/hook/useSaveSurvey.ts

```diff
@@ -12,8 +13,11 @@ export interface SaveState {
 export function useSaveSurvey(formType: SURVEY_FORM_TYPES, orgUnitId: Id, surveyId?: Id) {
     const { compositionRoot } = useAppContext();
     const [saveCompleteState, setSaveCompleteState] = useState<SaveState>();
+    const { currentHospitalForm } = useCurrentSurveys();
 
     const saveSurvey = (questionnaire: Questionnaire) => {
+        if (formType === "PPSWardRegister" || formType === "PPSPatientRegister")
+            orgUnitId = currentHospitalForm?.orgUnitId ?? "";
```

Comment:
Create a new variable instead of change current one

Reply:
done in https://github.com/EyeSeeTea/amr-surveys/pull/11